### ISSUE TITLE
[BUG] :: There appears to be a bug on Line 545 of operations.py #214

### ISSUE THREAD

**Issue by AlexaVillaume** [PYPIT/arclines#19]
(See also [https://github.com/PYPIT/PYPIT/issues/214](url))

I'm on the up-to date version of the master branch and I'm trying to reduce a set of data that I successfully reduced with an earlier version of PYPIT.

For my .pypit file I have,

```
# Reduce
bias useframe overscan
arc calibrate method arclines
trace slits number 1
```

I get the following bug,

```
Traceback (most recent call last):
  File "/Users/alexa/anaconda/bin/run_pypit", line 6, in <module>
    exec(compile(open(__file__).read(), __file__, 'exec'))
  File "/Users/alexa/PYPIT/bin/run_pypit", line 16, in <module>
    run_pypit.main(args)
  File "/Users/alexa/PYPIT/pypit/scripts/run_pypit.py", line 82, in main
    use_masters=args.use_masters, logname=logname, debug=debug)
  File "/Users/alexa/PYPIT/pypit/pypit.py", line 228, in PYPIT
    status = armlsd.ARMLSD(fitsdict)
  File "/Users/alexa/PYPIT/pypit/armlsd.py", line 150, in ARMLSD
    update = slf.MasterWaveCalib(fitsdict, sc, det)
  File "/Users/alexa/PYPIT/pypit/arsciexp.py", line 745, in MasterWaveCalib
    wv_calib = ararc.calib_with_arclines(self, det)
  File "/Users/alexa/PYPIT/pypit/ararc.py", line 554, in calib_with_arclines
    best_dict, final_fit = semi_brute(spec, aparm['lamps'], aparm['wv_cen'], aparm['disp'], fit_parm=aparm) #min_ampl=min_ampl,
  File "/Users/alexa/arclines/arclines/holy/grail.py", line 223, in semi_brute
    full_NIST = arcl_io.load_line_lists(lines, NIST=True)
  File "/Users/alexa/arclines/arclines/io.py", line 134, in load_line_lists
    line_lists = vstack(lists, join_type='exact')
  File "/Users/alexa/anaconda/lib/python2.7/site-packages/astropy/table/operations.py", line 253, in vstack
    out = _vstack(tables, join_type, col_name_map)
  File "/Users/alexa/anaconda/lib/python2.7/site-packages/astropy/table/operations.py", line 789, in _vstack
    out_descrs = get_descrs(arrays, col_name_map)
  File "/Users/alexa/anaconda/lib/python2.7/site-packages/astropy/table/operations.py", line 545, in get_descrs
    .format(names[0], tme._incompat_types))
astropy.table.np_utils.TableMergeError: The 'Ei           Ek' columns have incompatible types: ['string224', 'string224', 'string192', 'string216', 'string216', 'int64', 'string192', 'string200']
```

**Comment by profxj** [comment]
I may have caught it.  Try the lrisr_400_8500 branch again.

**Comment by AlexaVillaume** [comment]
I'm still having the same issue. What files are you using during your reductions?

**Comment by profxj** [comment]
I was running a simple test with arclines.  I'll send you an email to try it.

**Comment by profxj** [comment]
Actually do this:

1.  Get on the lrisr_400_8500 branch
2.  Go to arclines/arclines/holy/tests
3.  Do:  
``` python test_spec_to_sol.py '''

**Comment by profxj** [comment]
Well, what I want is for you to generate a test in this branch
which breaks the code in a manner similar to the Issue that
you have raised.  Then I'll be able to fix it.  Thanks!

**Comment by AlexaVillaume** [comment]
Fixed in pull request #20
